interpreter/cling/lib/Interpreter/IncrementalExecutor.h

```diff
@@ -153,8 +153,8 @@ namespace cling {
     };
 
     IncrementalExecutor(clang::DiagnosticsEngine& diags,
-                        const clang::CompilerInstance& CI,
-                        bool Verbose);
+                        const clang::CompilerInstance& CI, bool Verbose,
+                        bool Debug);
```

Comment:
Can we use `DebugInfoKind` in the CompilerInstance's `CodeGenOptions` instead of this flag?

Reply:
Done. I was not aware of this possibility. I kept the bool in the `IncrementalJIT`, because otherwise I needed to include the header for the `CodeGenOptions`.